web/src/lib.rs

```diff
@@ -156,6 +156,9 @@ pub struct Config {
     #[serde(rename = "logLevel")]
     log_level: log::Level,
 
+    #[serde(rename = "allowSwfDownload")]
+    allow_swf_download: bool,
```

Comment:
Is this really necessary? Since this config isn't used in Rust, I think it's best to not serialize it for no reason.

Reply:
It's mostly here because I wanted to match log_level and warn_on_unsupported_content, not necessarily in terms of functionality, but in terms of styling. Within the extension options, all the settings are snake_cased, but within the config, all the parameters are camelCased. I can't have both snake case and camel case refer to the same option without serde. So, if you want me to remove this, I have to ask , should I camelCase the extension settings or snake_case the configuration options?